src/Pmi/Evaluation/Evaluation.php

```diff
@@ -695,4 +676,28 @@ public function getReasonDisplayText()
         $reasonDisplayText = array_search($this->evaluation['eh_reason'], self::$cancelReasons);
         return !empty($reasonDisplayText) ? $reasonDisplayText : 'Other';
     }
+
+    private function addDiastolicBloodPressureConstraint($form, $field)
```

Comment:
While the only field where we currently use the `compare` option is diastolic blood pressure, there's nothing in this functionality that is specific to this measure. Can we rename this to something like `addCompareConstraint`?